src/plays/states/States.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
```

Comment:
Could you please drop a Readme file like this one to explain what is the project about and anything that a beginner looking into the code can take away?

https://github.com/atapas/react-play/blob/main/src/plays/social-card/Readme.md

Reply:
Added States.md

Reply:
@Abhishek-90 the file name should be Readme.md